### PR TITLE
Release 0.9.4

### PR DESCRIPTION
**Pull request by dwhswenson**
Release notes:

OpenPathSampling 0.9.4 includes a few new features, several important bug fixes, and a lot of minor improvements. A few highlights include:

* New ability to include velocities when creating an OPS trajectory from and MDTraj trajectory
* New default colors in visualizations (better visibility for the most common type of color blindness)
* Significant updates to the documentation
* Major improvements in Python 3 support
* Critical bug fix in the direct MD flux calculation
* Critical bug fix in two-way shooting

Details on all improvements below. 

## New features
* Add velocities as input to `trajectory_from_mdtraj` (#750)
* Tools for converting flux to `pandas` structures (#735)

## Bugs fixed
* Allow unicode for string options in engine option check (#757)
* Fix bug in `BackwardFirstTwoWayShooting` (#752)
* Fix problem with `HistogramPlotter2D` limits (#732)
* Fix bug in `snapshot.md box_vectors` (#731)
* Fix bug in `DirectSimulation` flux (#726)

## Miscellaneous improvements
* Default path tree colors are better for color-blind (#733)
* Improvements to documentation (#751, #748, #744, #737, #729, #723)
* Improved Python 3 compatibility (#743, #741, #739, #738)
* Improvements to examples (#746, #743, #719 [by @bolhuis])
* Behind-the-scenes upates to CI, testing, and deployment (#758, #749, #747, #745, #742)
* Dev install for simple developer installs (#728)
* Code readability updates (#724)

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -23,18 +23,18 @@ the development process on GitHub_, and follow `@pathsampling
...
 .. _GitHub: http://github.com/openpathsampling/openpathsampling
 
-.. note:: **Project status:** Currently we're at version 0.9.3.  There will
-          be some API cleanup before 1.0, but we don't expect the file
-          format to change.  Note, however, that files generated with Python
-          2 will not fully load with Python 3 or vice versa (partial Python 3
-          support starting in 0.9.3).
+.. note:: **Project status:** Currently we're at version 0.9.4.  There will
+          be some API cleanup before 1.0, but files created with current
+          this version will load in 1.0.  Note, however, that files
```
Comment (jhprinz) [inline docs/index.rst]: Typo: ... with current this version ...

**Comment by jhprinz** [comment]
Wow, that was quick!

**Comment by dwhswenson** [comment]
Thanks! Fixed the typo; as soon as this passes tests, I'll merge, then cut the GitHub release, then update the omnia recipe.

**Comment by dwhswenson** [comment]
> Wow, that was quick!

Happened to check my email just as that came in ;-)

**Pull request merged**